08-interop/test_spec08.py

```diff
@@ -59,22 +58,17 @@ def test_task01(riot_ctrl, log_nodes):
     'nodes', [pytest.param(['nrf52840dk', 'nrf52840dk'])], indirect=['nodes']
 )
 def test_task03(riot_ctrl):
-    # run `./compile_contiki.sh` relative to this file
-    subprocess.check_call(
-        ["./compile_contiki.sh"],
-        cwd=os.path.dirname(os.path.realpath(__file__)),
-    )
```

Comment:
Do we want to remove `./compile_contiki.sh` then or is this just meant as a temporary workaround for some non-RIOT build failures? In the latter case, I would propose to add a comment stating this here.

Reply:
Yes we want to.  We have the binary already and pulling the contiki docker image causes the GH actions runner to fill up.  If you run it on your local machine it should still be ok, which is why we leave it in (in case we want to update the binary).